addons/mail_bot/static/src/messaging/component/messaging_menu/messaging_menu_tests.js

```diff
@@ -41,8 +41,12 @@ QUnit.test('rendering with OdooBot has a request (default)', async function (ass
             }
             return this._super(...arguments);
         },
-        'window.Notification': {
-            permission: 'default',
+        env: {
```

Comment:
should be defined above `mockRPC` (alphabetical order of obj keys)

Reply:
done